nodes/pods/nodes-pods-autoscaling-custom.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: ASSEMBLY
+:context: nodes-pods-autoscaling-custom
+[id="nodes-pods-autoscaling-custom"]
+= Automatically scaling pods based on custom metrics
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As a developer, you can use the custom metrics autoscaler to specify how {product-title} should automatically increase or decrease the number of pods for a deployment, stateful set, custom resource, or job based on custom metrics that are not just based on CPU or memory. 
+
+[NOTE]
+====
+{product-title} curently supports only the Prometheus trigger, which can use the installed {product-title} monitoring or an external Prometheus server as the metrics source.  
```

Comment:
`{product-title} curently supports only the Prometheus trigger` -> this is rendered as  `OpenShift Container Platform curently supports only the Prometheus trigger`

But should probably be:
`The the custom metrics autoscaler curently supports only the Prometheus trigger`